package.json

```diff
@@ -42,7 +42,8 @@
     "json-schema": "^0.4.0",
     "ansi-regex": "^5.0.1",
     "kubernetes-client/@kubernetes/client-node": ">=0.22.2",
-    "tough-cookie": "^4.1.3"
+    "tough-cookie": "^4.1.3",
+    "openid-client": "^4.9.0"
```

Comment:
Which deps are requiring it?
I see that it was present twice, should we not only upgrade lower version but also downgrade newer one? How safe is that?

<img width="1224" height="584" alt="Image" src="https://github.com/user-attachments/assets/6f98271b-50f4-4072-b177-943d9c2463bf" />